addon/components/frost-popover.js

```diff
@@ -367,8 +369,15 @@ export default Component.extend(PropTypeMixin, {
           marginLeft: -delta.left - arrowMargin,
           marginTop: -delta.top - arrowMargin
         })
+
+        if (isPresent(this.get('onDisplay'))) {
+          this.get('onDisplay')()
+        }
       } else {
         this.unregisterClickOff()
+        if (isPresent(this.get('onHide'))) {
```

Comment:
@helrac This should have been

`if (isPresent(this.get('onHide'))) && typeOf(this.get('onHide')) === 'function') {`